students/laura-hernandez/ejercicio-1/src/index.ts

```diff
@@ -0,0 +1,47 @@
+const recursiveFibonacci = (n: number) => {
+  let fib: number;
+  let fibMin1: number = 1;
+  let fibMin2: number = 0;
+
+  if (n < 0) {
```

Comment:
Attempts to use defensive programming by preventing the growth of cyclomatic complexity.